resource_customizations/iam.cnrm.cloud.google.com/IAMPolicyMember/testdata/healthy_upToDate.yaml

```diff
@@ -1,77 +0,0 @@
-apiVersion: iam.cnrm.cloud.google.com/v1beta1
```

Comment:
Looks like the contents were deleted, but the file was not. Is that the intention?

Reply:
Not sure what you mean? The file `iam.cnrm.cloud.google.com/IAMPolicyMember/testdata/healthy_upToDate.yaml` does not exist.
https://github.com/Kerwood/argo-cd/tree/health-check/google-cloud/resource_customizations/iam.cnrm.cloud.google.com/IAMPolicyMember/testdata

Reply:
Ah it's just GitHub's rendering confusing me then. Thanks!